slf4j-api/pom.xml

```diff
@@ -21,6 +21,19 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>animal-sniffer-maven-plugin</artifactId>
+        <configuration>
+          <ignores>
+            <ignore>org.slf4j.impl.StaticMDCBinder</ignore>
```

Comment:
Comment is on the commit but I see your point that it would be useful in the file.  I'll add that to this as soon as I get a chance.

--- Original Message ---

From: "Matt Bishop" <notifications@github.com>
Sent: April 27, 2015 10:18 PM
To: "qos-ch/slf4j" <slf4j@noreply.github.com>
Cc: "Jeremy Landis" <jeremylandis@hotmail.com>
Subject: Re: [slf4j] Apply Standard Maven Usage (#107)

> @@ -21,6 +21,19 @@
>    <build>
>      <plugins>
>        <plugin>
> +        <groupId>org.codehaus.mojo</groupId>
> +        <artifactId>animal-sniffer-maven-plugin</artifactId>
> +        <configuration>
> +          <ignores>
> +            <ignore>org.slf4j.impl.StaticMDCBinder</ignore>

Why are these ignored? Perhaps add a comment for each.

---
Reply to this email directly or view it on GitHub:
https://github.com/qos-ch/slf4j/pull/107/files#r29209860